src/rendervulkan.cpp

```diff
@@ -3156,6 +3156,9 @@ std::shared_ptr<CVulkanTexture> vulkan_acquire_screenshot_texture(uint32_t width
 
 float g_flLinearToNits = 400.0f;
 float g_flNitsToLinear = 1.0f / 100.0f;
+uint32_t g_bHDRItmEnable = 0;
```

Comment:
bool

Reply:
I tried using bool but couldn't get it working setting the Atom with xprop
`xprop -root -f GAMESCOPE_HDR_ITM_ENABLE 32b -set GAMESCOPE_HDR_ITM_ENABLE 0`
should work but it doesn't from my testing
8b and 16b don't work either
switching it to uint32_t made it work
any suggestions?

Reply:
hmm I can't get this to work with a bool
`DISPLAY=:1 xprop -root -f GAMESCOPE_HDR_ITM_ENABLE 32b -set GAMESCOPE_HDR_ITM_ENABLE False`
`DISPLAY=:1 xprop -root -f GAMESCOPE_HDR_ITM_ENABLE 16b -set GAMESCOPE_HDR_ITM_ENABLE False`
`DISPLAY=:1 xprop -root -f GAMESCOPE_HDR_ITM_ENABLE 8b -set GAMESCOPE_HDR_ITM_ENABLE False`
`DISPLAY=:1 xprop -root -f GAMESCOPE_HDR_ITM_ENABLE 32c -set GAMESCOPE_HDR_ITM_ENABLE 0`
all don't work if it's a bool :(

Reply:
now works as a bool by changing ItmEnable to be a spec constant